src/core/core.controller.js

```diff
@@ -985,8 +1020,10 @@ helpers.extend(Chart.prototype, /** @lends Chart */ {
 		// Find Active Elements for hover and tooltips
 		if (e.type === 'mouseout') {
 			me.active = [];
+			me._lastEvent = null;
 		} else {
 			me.active = me.getElementsAtEventForMode(e, hoverOptions.mode, hoverOptions);
+			me._lastEvent = e;
```

Comment:
If `'click'` event is picked here, [this issue](https://github.com/chartjs/Chart.js/pull/5817#issuecomment-440978119) will occur.

Reply:
I cannot reproduce that issue in the codepen I have provided with the PR code, I think that issue there was with refiring the actual click event.

Reply:
I see this issue when clicking the legend in [your codepen](https://codepen.io/Evertvdw/pen/PgqbWq).